include/config/overworld.h

```diff
@@ -3,6 +3,7 @@
 
 // Movement config
 #define OW_RUNNING_INDOORS          GEN_LATEST  // In Gen4+, players are allowed to run indoors.
+#define OW_AUTO_SIGNPOST            FALSE       // When enabled, if the tile that the player is facing has MB_SIGNPOST, MB_POKEMART_SIGN, or MB_POKEMON_CENTER_SIGN, the player will automatically read the signpost.
```

Comment:
I wonder if we should include which other games do this (FRLG for sure, but maybe Gen4/Gen5 too?)

Reply:
Are you saying

1) Make this a generation config
2) In the description, note which games do this
3) Both

I don't have a good way to test all the main series games. I can make it a priority for the future, just won't happen soon.

Reply:
I think 1 is not possible because it's not consistent within generations, so rather 2. I think a simple "as in FRLG" would suffice for now though, and other games can be tested later well after this PR's merger

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/fb53297e644a069065b433d343724df4553a3d9d